### ISSUE TITLE
Error since downshift 6.0.12

### ISSUE THREAD

**Issue by hk86** [downshift-js/downshift#1214]
Hi there,

my production build produces an error since downshift 6.0.12. On 6.0.11 I do not see this error:

```
react-dom.production.min.js:216 TypeError: r is not a function
    at M (downshift.esm.js:373)
    at k (downshift.esm.js:352)
    at ce (downshift.esm.js:3125)
    at downshift.esm.js:1829
    at Object.uo [as useReducer] (react-dom.production.min.js:159)
    at t.useReducer (react.production.min.js:23)
    at L (downshift.esm.js:1836)
    at downshift.esm.js:3069
    at se (downshift.esm.js:3245)
    at A (Search.jsx:66)
```

It this code here:

```
function getNextNonDisabledIndex(moveAmount, baseIndex, itemCount, getItemNodeFromIndex, circular) {
  var currentElementNode = getItemNodeFromIndex(baseIndex); <------------ THIS LINE

  if (!currentElementNode || !currentElementNode.hasAttribute('disabled')) {
    return baseIndex;
  }

  if (moveAmount > 0) {
    for (var index = baseIndex + 1; index < itemCount; index++) {
      if (!getItemNodeFromIndex(index).hasAttribute('disabled')) {
        return index;
      }
    }
  } else {
    for (var _index = baseIndex - 1; _index >= 0; _index--) {
      if (!getItemNodeFromIndex(_index).hasAttribute('disabled')) {
        return _index;
      }
    }
  }

  if (circular) {
    return moveAmount > 0 ? getNextNonDisabledIndex(1, 0, itemCount, getItemNodeFromIndex, false) : getNextNonDisabledIndex(-1, itemCount - 1, itemCount, getItemNodeFromIndex, false);
  }

  return -1;
}
```

**Comment by meotimdihia** [comment]
Something was broke since 6.0.12. Can't type characters in the input element:
```javascript         <input {...getInputProps({})} />```

**Comment by meotimdihia** [comment]
Something was broke since 6.0.12. Can't type characters in the input element:

<input {...getInputProps({})} />

**Comment by silviuaavram** [comment]
Hi! Sorry for the issue, I will figure it out. I’d rather find a fix and
not revert, since 6.0.12 has been an improvement in bundle size and I’d
like to keep that. Will update on the status. Meanwhile just stay on 6.0.11
until this problem is solved. Thank you!

**Comment by silviuaavram** [comment]
@meotimdihia @hk86 can you help out with usage examples? I don't know your use cases and the error is not obvious as far as I first checked.

**Comment by meotimdihia** [comment]
This is an example: 
https://codesandbox.io/s/eager-andras-s21ie?file=/pages/autocomplete.js

But the bug is just happening on the production build. You must create a new terminal and run the command: `yarn build & yarn start`
![chrome_2021-01-10_17-47-49](https://user-images.githubusercontent.com/300961/104120781-22c76b80-536c-11eb-8a41-4efb4f6506a8.png)

goto page on port 3001 to see the production page. And try to type character on input.

![chrome_2021-01-10_17-52-38](https://user-images.githubusercontent.com/300961/104120860-a5e8c180-536c-11eb-9471-925f480988a4.png)

**Comment by silviuaavram** [comment]
<img width="1445" alt="Screenshot 2021-01-10 at 13 10 02" src="https://user-images.githubusercontent.com/11275392/104121199-2abfe500-5345-11eb-8067-47b2bd74d5ad.png">

**Comment by meotimdihia** [comment]
@silviuaavram it's strange. It doesn't work for me on any browser, even with firefox. 
Once, I downgrade the Downshift version to 6.0.11 then it works.

**Comment by mattleonowicz** [comment]
I can confirm the same thing. I'd suggest reverting. Breaking stuff for unaware people doing upgrades is worse then some bundle size savings

**Comment by silviuaavram** [comment]
Makes sense.

In the meantime can anyone help me out debugging this thing? With revert or not, the final version is to have the changes + the error fixed.

Thanks!

**Comment by silviuaavram** [comment]
So, the PR is up. Will merge it soon.

If you are able to repro this and share the repro steps / environment please do so. I look forward to also have it fixed. Thanks!

**Comment by silviuaavram** [comment]
Found the issue, working on a fix.

**Comment by silviuaavram** [comment]
Fix done. Released 6.0.15-alpha which is 6.0.12 but without breaking anything.

Can you update to this version and do a smoke test with your usages, just to be super-duper sure?

Thank you and sorry for the inconvenience!

**Comment by mattleonowicz** [comment]
It works in my case
